### PR TITLE
Update operatorhub test due to new bundle format

### PR DESCRIPTION
**Pull request by mmajerni**

### CODE REVIEW

File: src/test/java/apicurito/tests/configuration/TestConfiguration.java [hunk 1]
```diff
@@ -52,6 +53,7 @@
     public static final String QUAY_NAMESPACE = "apicurito.config.quay.namespace";
     public static final String QUAY_TOKEN = "apicurito.config.quay.auth.token";
     public static final String QUAY_OPSRC_TOKEN = "apicurito.config.quay.opsrc.token";
+    public static final String QUAY_PULL_SECRET = "apicurito.config.quay.pull.secret";
```
Comment (jsafarik) [inline src/test/java/apicurito/tests/configuration/TestConfiguration.java:56]: is this some other pull secret than https://github.com/Apicurio/apicurio-qe/pull/61/files#diff-b81094bfde713d8bd4bb083970afdbff5a188b45e4cd3d20c95a89c5731570d4R46 ?

Comment (jsafarik) [reply src/test/java/apicurito/tests/configuration/TestConfiguration.java:56]: the link might not be working that well, I mean the pull secret on line #46 (APICURITO_PULL_SECRET)?

Comment (mmuzikar) [reply src/test/java/apicurito/tests/configuration/TestConfiguration.java:56]: The value itself is definitely different. It is used for pulling bundle resources - which required some openshift images and also our custom bundle images (if we ever decide to make them private). This pull secret can be obtained from [https://cloud.redhat.com/openshift/install/crc/installer-provisioned](https://cloud.redhat.com/openshift/install/crc/installer-provisioned)

Comment (mmajerni) [reply src/test/java/apicurito/tests/configuration/TestConfiguration.java:56]: yes as Martin said there are two different pull secrets for the quay.

Comment (jsafarik) [reply src/test/java/apicurito/tests/configuration/TestConfiguration.java:56]: ok, thanks for the clarification

File: src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java [hunk 2]
```diff
@@ -86,106 +87,65 @@ public void checkThatApicuritoImageIs(String podType, String value) {
...
+        } catch (IOException | TimeoutException | InterruptedException e) {
+            e.printStackTrace();
         }
+    }
 
+    private static OpenShiftService getOpenShiftService(String quayProject) {
         OpenShiftUser adminUser = new OpenShiftUser(
             TestConfiguration.openshiftUsername(),
-            TestConfiguration.openshiftPassword(),
+            TestConfiguration.openshiftUsername(),
```
Comment (jsafarik) [inline src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java:133]: a little confused why username is here twice instead of username and password?

**Comment by jsafarik** [comment]
otherwise lgtm, but as I don't know much about operatorhub testing, someone else should check it as well.

File: src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java (hunk 2 above)
Comment (mmajerni) [reply src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java:133]: lucky mistake :) you are right should be there password instead of username.

**Review by mmuzikar — APPROVED** [review]
lgtm

**Pull request merged**